Tests/Test_DisplayApps.c

```diff
@@ -126,7 +126,7 @@ void Task1(void *arg)
 
     testPercentageComp(&UpdateDisplay_SetAccel);
 
-    error = Display_Fault(OSErrLocBitmap, FaultBitmap);
+    error = Display_Error(OSErrLocBitmap, FaultBitmap);
```

Comment:
Does FaultBitmap still exist?

Reply:
It doesn't. A lot of the tests were outdated and still used it, but I chose to only partially update only this test file when I renamed Display_Fault to Display_Error. Should I go ahead and finish updating this test, or would it be better to just take it off the PR for now?

Reply:
Let's finish updating the test.

Reply:
For places where the test uses assertDisplayError, I'm guessing we would want to replace this with assertUpdateDisplayError. However, the test can't access assertUpdateDisplayError because it's a static function. Would it be best to make assertUpdateDisplayError a public function, add test macros, or do something else entirely?

Reply:
Also, I'm guessing the call to Display_Error is meant to test the function? 

`error = Display_Error(OSErrLocBitmap, (error_code_t)error); // Testing Display_Error?
assertUpdateDisplayError(error == DISPLAY_ERR_NONE ? UPDATEDISPLAY_ERR_NONE : UPDATEDISPLAY_ERR_DRIVER);
OSTimeDlyHMSM(0, 0, 3, 0, OS_OPT_TIME_HMSM_STRICT, &e);`

I chose to replace Fault_Bitmap with whatever the display error was before on the grounds that it didn't matter as long as there was something there for the code, but I wanted to check if that's actually okay